packages/openai/src/RPA/OpenAI.py

```diff
@@ -0,0 +1,241 @@
+
+        Robot Framework example:
+
+        .. code-block:: robotframework
+
+            ${response}  Completion Create    
+            ...     Write a tagline for an icecream shop.  
+            ...     temperature=0.6
+            Log     ${response}
+
+        Python example:
+
+        .. code-block:: python
+
+            result = baselib.completion_create(
+                'Create a tagline for icecream shop',
+                temperature=0.6,
+            )
+            print(result)
+
+        """
+        response = openai.Completion.create(
+            model=model,
+            prompt=prompt,
+            temperature=temperature,
+            max_tokens=max_tokens,
+            top_p=top_p,
+            frequency_penalty=frequency_penalty,
+            presence_penalty=presence_penalty
+        )
+        self.logger.info(response)
+        if result_format == "string":
+            text = response['choices'][0]['text'].strip()
+            return text
+        if result_format == "json":
+            return response
+        else:
+            return None
+
+    @keyword
+    def image_create(self, 
+        prompt: str, 
+        size: Optional[str] = "512x512", 
+        n: Optional[int] = 1,
+        result_format: Optional[str] = "list"
+        ) -> None:
+        """Keyword for creating one or more images in OpenAI. Keyword returns a list of urls for the images created.
+        
+        :param size: Size of the files to be created. 256x256, 512x512, 1024x1024
+
+        :param prompt: Text prompt/description of the image(s) to be created. 
+
+        :param n: Number of images. 
+
+        :param result_format: Result format (list / json).
+
+        Robot Framework example:
+
+        .. code-block:: robotframework
+ 
+            ${images}    Image Create  
+            ...   Cartoon style picture of a cute monkey skateboarding.  
+            ...   size=256x256  
+            ...   n=2 
+            FOR    ${url}    IN    @{images}
+                Log    ${url}
+            END
+        
+        Python example:
+
+        .. code-block:: python
+
+            images = baselib.image_create(
+                'Cartoon style picture of a cute monkey skateboarding',
+                size='256x256',
+                n=2,
+            )
+            for url in images:
+                print(url)
+
+        """
+        response = openai.Image.create(prompt=prompt, size=size, n=n)
+        self.logger.info(response)
+        urls = []
+        if result_format == "list":
+            for _url in response['data']:
+                urls.append(_url['url'])
+                self.logger.info(_url)
+            return urls 
+        if result_format == "json":
+            return response
+        else:
+            return None
+
+    @keyword
+    def image_create_variation(self,
+        src_image: str,
+        size: Optional[str] = "512x512", 
+        n: Optional[int] = 1,
```

Comment:
commented on this in previous keywords

Reply:
...the irony is that `n` comes directly from OpenAI API :-)) fixed